wp-graphql.php

```diff
@@ -212,6 +214,22 @@ private function filters() {
 
 		}
 
+		public function activate() {
+			flush_rewrite_rules();
+			// Save the version of the plugin as an option in order to force actions
+			// on upgrade.
+			if ( false === get_option( 'wp_graphql_version', false ) ) {
+				add_option( 'wp_graphql_version', WPGRAPHQL_VERSION, null, 'no' );
```

Comment:
I don't think you have to actually check if the option exists already. You can just use `update_option` since it falls back to `add_option` if the option doesn't exist.